grpc/flatbuffers-js-grpc/src/greeter_grpc.d.ts

```diff
@@ -1,54 +1,55 @@
 // Generated GRPC code for FlatBuffers TS *** DO NOT EDIT ***
 import { flatbuffers } from 'flatbuffers';
-import *  as Greeter_fbs from './greeter_generated';
+import { HelloReply as Welcome_HelloReply } from './welcome/hello-reply';
+import { HelloRequest as Welcome_HelloRequest } from './welcome/hello-request';
 
 import * as grpc from 'grpc';
```

Comment:
Is this okay to do? since I am not really sure how the new imports actually work